Update aws-connector-settings-reference.md (#4710)
harness/developer-hub: docs/platform/connectors/cloud-providers/ref-cloud-providers/aws-connector-settings-reference.md

https://www.loom.com/share/2f02907ff84247acaf3e617c05acab34-->
<docvideo src="https://www.loom.com/share/2f02907ff84247acaf3e617c05acab34" />

### FAQ

> #### What are the components that are needed to set up the EKS Authentication Feature in AWS and Harness?

Reviewer: ```suggestion
#### What are the required components to set up the EKS Authentication feature in AWS and Harness?
```

**In Harness**

> 1. AWS Connector

Reviewer: ```suggestion
1. AWS Connector.
```

> 2. Instal AWS IAM Authenticator on the Delegate via INIT_SCRIPT

Reviewer: ```suggestion
2. Instal AWS IAM Authenticator on the delegate via `INIT_SCRIPT`.
```

**In AWS**

> 1. IAM Role in AWS Account with the correct permissions

Reviewer: ```suggestion
1. An IAM Role in the AWS account with the correct permissions.
```

> 2. Configure a Kubernetes Service Account in the EKS cluster 

Reviewer: ```suggestion
2. Configure a Kubernetes Service Account in the EKS cluster.
```
